client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -1,10 +1,10 @@
 import React, { Component } from 'react';
 import SingleAppContainer from './SingleAppContainer.js';
-import {topFree, topGrossing, topPaid, trending} from './mockData.js';
 import './TopChartsStyle.css';
 import '../styles.css';
  
 class TopCharts extends Component {
```

Comment:
Can you also add PropTypes for this. At the minimum at least define data is a required object